src/plugins/Makefile.am

```diff
@@ -128,9 +129,18 @@ libgs_plugin_dpkg_la_CFLAGS = $(GS_PLUGIN_CFLAGS) $(WARN_CFLAGS)
 if HAVE_EXTERNAL_APPSTREAM
 libgs_plugin_external_appstream_la_SOURCES =		\
 	gs-plugin-external-appstream.c
-libgs_plugin_external_appstream_la_LIBADD = $(GS_PLUGIN_LIBS)
+libgs_plugin_external_appstream_la_LIBADD = $(GS_PLUGIN_LIBS) $(SOUP_LIBS)
```

Comment:
Correction, this should be part of the next commit. You should also add `$(SOUP_CFLAGS)` to `libgs_plugin_external_appstream_la_CFLAGS` there.